glide.yaml

```diff
@@ -6,7 +6,7 @@ owners:
   email: shaun@tigera.io
 import:
 - package: github.com/coreos/etcd
-  version: ^3.2.0-rc.1
+  version: 1e1dbb23924672c6cd72c62ee0db2b45f778da71
```

Comment:
Pinning to the same version as was in glide.lock. Updating caused some dependency conflicts.